src/main/java/com/flightstats/hub/cluster/DistributedLeadershipLockManager.java

```diff
@@ -0,0 +1,62 @@
+package com.flightstats.hub.cluster;
+
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.recipes.locks.InterProcessSemaphoreMutex;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.concurrent.TimeUnit;
+
+public class DistributedLeadershipLockManager {
+    private final static Logger logger = LoggerFactory.getLogger(DistributedLeadershipLockManager.class);
```

Comment:
@Slf4j? Since we started replacing log statements with lombok annotation as part of refactor, we shud continue doing it.

Reply:
Whoops, I thought I went back and changed all those. 👍